engine/src/main/java/org/hibernate/validator/internal/engine/ConfigurationImpl.java

```diff
@@ -81,16 +81,17 @@
 	private ValidationProviderResolver providerResolver;
 	private final ValidationBootstrapParameters validationBootstrapParameters;
 	private boolean ignoreXmlConfiguration = false;
-	private final Set<InputStream> configurationStreams = CollectionHelper.newHashSet();
```

Comment:
slipped through during the merge. There were a couple of things which were "odd", eg some pom changes. I thought I caught most of it. Apparently not.